backend/tests/test_action_engine.py

```
@@ -371,7 +371,36 @@ def test_cc_candidate_href_carries_context(self):
-        assert "cost_basis=17240.0" in href
+        # cost_basis is emitted PER-SHARE (issue #186) — scanner expects that
+        # unit. 17240.0 total / 100 shares = 172.4 per share.
+        assert "cost_basis=172.4" in href
+        # Must never emit the raw total — that breaks the 10% rule downstream.
+        assert "cost_basis=17240" not in href
+
+    def test_cc_candidate_href_emits_per_share_basis_canonical_f_case(self):
+        """Regression for #186: F at 100 shares with broker_cost_basis=$1,320.66
+        must emit cost_basis=13.2066 per-share, not the raw total.
+
+        Pre-fix the scanner's 10%-rule floor became 1320.66 × 1.01 ≈ $1,333,
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)